readthedocs/templates/projects/project_delete.html

```diff
@@ -6,6 +6,16 @@
 {% block content-header %}<h1>{% blocktrans with project.name as name %}Delete {{ name }}?{% endblocktrans %}</h1>{% endblock %}
 
 {% block content %}
+
+  {% if is_superproject %}
+    {% url "projects_subprojects" project.slug as subproject_url %}
+    <p>
+      {% blocktrans %}
+        This project <a href="{{ subproject_url }}">has subprojects</a> under it. Deleting this will make them as regular projects. This will effect the URLs of the subprojects and they will be served normally as other projects.
+      {% endblocktrans %}
```

Comment:
You should use the blocktrans like this https://github.com/rtfd/readthedocs.org/blob/30583646e470b1bdcf606c52e851835d6495df20/readthedocs/templates/wipe_version.html#L16-L18

(also, splitting the line will make this easier to read in the code)

Reply:
@stsewd 
we cannot do the url lookup in this way here.
 
> Reverse URL lookups cannot be carried out within the blocktrans

Source: https://docs.djangoproject.com/en/2.2/topics/i18n/translation/#blocktrans-template-tag